### PR TITLE
Rename culprits into saviours

### PR DESCRIPTION
**Pull request by AndreiVidrasco**

### CODE REVIEW

File: src/main/java/jenkins/plugins/office365connector/FactsBuilder.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
     final static String NAME_STATUS = "Status";
     final static String NAME_REMARKS = "Remarks";
-    final static String NAME_CULPRITS = "Culprits";
+    final static String NAME_SAVIOURS = "Saviours";
```
Comment (damianszczepanik) [inline src/main/java/jenkins/plugins/office365connector/FactsBuilder.java]: why?

**Comment by damianszczepanik** [comment]
This is not the matter of culture - this is the matter how whole Jenkins ecosystem names this type of event/behave.
Also this is using finder instead of thief because he can return it back.

**Comment by AndreiVidrasco** [comment]
I'm sorry, but if someone long time ago had a different culture and named like that in jenkins ecosystem, doesn't mean we can't change it nor it means we should continue using. I'm not using jenkins actively in my work, but if you can point out where to submit same change for jenkins, I will happily do.

About your second remark with finder/thief, I'm not sure I understand your point nor how it's related to this question.

**Comment by Sedwick** [comment]
Why not split the difference and call them a "suspect"? 😜 I understand both points here, but it usually helps to use standard nomenclature.

**Comment by damianszczepanik** [comment]
'suspect' sounds much better

**Comment by casz** [comment]
@damianszczepanik perhaps allow user freedom? Make it configurable with a sane default?

**Comment by damianszczepanik** [comment]
no way

**Comment by casz** [comment]
Why not? minor price to pay for user choice?

**Comment by damianszczepanik** [comment]
The price is paid when configuration must be maintained and when the user expects to have the plugin as simple as possible with the powerful at the same time.

I don't see the value of this unless you give me some reference for other plugin which are not less popular than this one :)

**Comment by casz** [comment]
I'd be happy to create another plugin called `less-opinionated-office-365-connector-plugin` for this reason alone if I went back to using Microsoft teams, lucky I don't. @Sedwick and @AndreiVidrasco you are free to fork the plugin and make the necessary adjustment.

**Comment by AndreiVidrasco** [comment]
@damianszczepanik I don't look for plugins all day to find one which you will like.
I'm an iOS Developer and I use Xcode all day. It's fairly to say that Xcode is far more popular than this plugin.
git's command "git blame", which is using same blaming pattern like this plugin, inside of Xcode was mapped to Authors and that was pretty recently, around 2 years ago.
Here is a blog post explaining why it has been done
https://jamesdempsey.net/2017/10/17/xcode-whos-to-blame/

**Comment by damianszczepanik** [comment]
Well while "culprits" has negative meeting (however I've never though about it that way) the "saviours" is completely the opposite and I believe there are only a couple developers who can properly predict what does it stand for when they see such label at the application menu.

Authors seems to be reasonable, will use that name.

**Comment by casz** [comment]
Good enough for me

**Comment by AndreiVidrasco** [comment]
I put saviours as exact opposite to culprits, but have no hard feelings over this term.
Authors sounds good to me

**Comment by damianszczepanik** [comment]
Based on https://javadoc.jenkins.io/jenkins/scm/RunWithSCM.html#getCulprits-- it should be named Committers

**Pull request closed without merging**